lib/msal-core/src/ServerRequestParameters.ts

```diff
@@ -54,7 +54,11 @@ export class ServerRequestParameters {
   constructor (authority: Authority, clientId: string, scope: Array<string>, responseType: string, redirectUri: string, state: string) {
     this.authorityInstance = authority;
     this.clientId = clientId;
-    this.scopes = scope;
+    if (!scope) {
```

Comment:
can we add a test to make sure that what we are fixing is fixed

Reply:
Added tests